packages/ERTP/src/paymentLedger.js

```diff
@@ -334,7 +334,7 @@ export const preparePaymentLedger = (
       withdrawInternal,
     }),
     recoverySetsState,
-    recoverySetsState === 'noRecoverySets' ? undefined : paymentRecoverySets,
+    paymentRecoverySets,
```

Comment:
Perhaps I just got the condition reversed, so a narrower fix would be the following?

```suggestion
    recoverySetsState === 'noRecoverySets' ? paymentRecoverySets : undefined,
```

OTOH, having such a conditional in the caller at all makes the cross-coupling worse than your correction, so no change suggested.

Reply:
Since still "no change suggested", resolving.